Adding GroupViT Models (#17313)
huggingface/transformers: docs/source/en/model_doc/groupvit.mdx

> <!--Copyright 2021 The HuggingFace Team. All rights reserved.

Reviewer: ```suggestion
<!--Copyright 2022 The HuggingFace Team. All rights reserved.
```

Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
the License. You may obtain a copy of the License at

http://www.apache.org/licenses/LICENSE-2.0

Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
specific language governing permissions and limitations under the License.
-->

# GroupViT

## Overview

The GroupViT model was proposed in [GroupViT: Semantic Segmentation Emerges from Text Supervision](https://arxiv.org/abs/2202.11094) by Jiarui Xu, Shalini De Mello, Sifei Liu, Wonmin Byeon, Thomas Breuel, Jan Kautz, Xiaolong Wang.

> GroupViT is a vision-language model that could perform zero-shot semantic segmentation on any given vocabulary categories.

Reviewer: ```suggestion
Inspired by [CLIP](clip), GroupViT is a vision-language model that can perform zero-shot semantic segmentation on any given vocabulary categories.
```

> Inspired by [CLIP](clip), GroupViT is a vision-language model that can perform zero-shot semantic segmentation on any given vocabulary categories.

Reviewer 2: the url is missing for clip.

Reviewer: This will automatically link to the docs of CLIP ;)

The abstract from the paper is the following:

*<INSERT PAPER ABSTRACT HERE>*

Tips:

<INSERT TIPS ABOUT MODEL HERE>

This model was contributed by [INSERT YOUR HF USERNAME HERE](https://huggingface.co/<INSERT YOUR HF USERNAME HERE>).

> The original code can be found [here](<INSERT LINK TO GITHUB REPO HERE>).

Reviewer: To be filled in

Tips:

> - You may specify `output_segmentation=True` to get the segmentation logits of input texts. 

Reviewer: ```suggestion
- You may specify `output_segmentation=True` in the forward of `GroupViTModel` to get the segmentation logits of input texts. 
```

> - The quickest way to get started with GroupViT is by checking the [example notebooks](https://github.com/xvjiarui/GroupViT/blob/main/demo/GroupViT_hf_inference_notebook.ipynb) (which showcase zero-shot segmentation inference). One can also check out the [huggingface space demo](https://huggingface.co/spaces/xvjiarui/GroupViT) to play with GroupViT. 

Reviewer: ```suggestion
- The quickest way to get started with GroupViT is by checking the [example notebooks](https://github.com/xvjiarui/GroupViT/blob/main/demo/GroupViT_hf_inference_notebook.ipynb) (which showcase zero-shot segmentation inference). One can also check out the [HuggingFace Spaces demo](https://huggingface.co/spaces/xvjiarui/GroupViT) to play with GroupViT. 
```
